tests/Feature/ThrowingSupervisor.php

```
@@ -0,0 +1,17 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Exceptions\SSHCommandError;
+use App\Models\ServerLog;
+use App\Services\ProcessManager\Supervisor;
+
+class ThrowingSupervisor extends Supervisor
+{
+    public function restartMany(array $ids, ?int $siteId = null): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$ids'. (undefined)

(UnusedFormalParameter)